deployer/scripts/util.sh

```diff
@@ -14,8 +14,8 @@ function generate_JKS_chain() {
         extension_names="${extension_names},dns:${name}"
     done
 
-    if [ "$ADD_OID" = true ]; then
-        extension_names="${extension_names},oid:1.2.3.4.5.5"
+    if [ ! -z "$OPENSHIFT_OID// " ]; then
```

Comment:
@jcantrill yes - if `-n` is true it means arg is defined and is not empty - this is the preferred way to check rather than using `! -z`.